api/src/wfl/module/covid.clj

```diff
@@ -859,18 +869,33 @@
            first
            :count))))
 
-(defn ^:private terra-executor-workflows
-  [tx {:keys [workspace details] :as _executor}]
-  (when-not (postgres/table-exists? tx details)
-    (throw (ex-info "Missing executor details table" {:table details})))
+(defn ^:private terra-workflows-from-records
+  [{:keys [workspace] :as _executor} records]
   (letfn [(from-record [{:keys [workflow submission status] :as record}]
             (combine-record-workflow-and-outputs
              record
              (firecloud/get-workflow workspace submission workflow)
              (when (= "Succeeded" status)
                (firecloud/get-workflow-outputs workspace submission workflow))))]
-    (let [query "SELECT * FROM %s WHERE workflow IS NOT NULL ORDER BY id ASC"]
-      (map from-record (jdbc/query tx (format query details))))))
+    (map from-record records)))
+
+(defn ^:private terra-executor-workflows
+  [tx {:keys [details] :as executor}]
+  (when-not (postgres/table-exists? tx details)
+    (throw (ex-info "Missing executor details table" {:table details})))
+  (let [query "SELECT * FROM %s WHERE workflow IS NOT NULL ORDER BY id ASC"]
+    (->> (jdbc/query tx (format query details))
+         (terra-workflows-from-records executor))))
```

Comment:
FYI: Some linter doesn't like 2-form `->` and `->>` macros.

Reply:
Adding value then!

Reply:
I don't care either way, really, but it will cut down on linter noise.